data/anaconda_options.txt

```diff
@@ -258,3 +258,7 @@ Enable legacygrub (for installing older OSes). You probably do not want this.
 waitfornet
 Wait for network connectivity at the beginning of the installation with the
 timeout in seconds specified as a mandatory value of the option.
+
+decorated
+Run GUI installer in a decorated window. By default, the window is not decorated, so it doesn't have a title bar,
+resize controls, etc.
```

Comment:
A similar note should be also added to the boot option docs (`docs/boot-options.rst`) so that the new boot option also shows up in the Anaconda documentation on Read The Docs.

(I guess in the future it might make sense to generate that document from argparse output the same way Pykickstart does it for kickstart commands, so we don't need to keep multiple places in sync.)

Reply:
Fixed.